### PR TITLE
Fix no camera alert spam

### PR DESCRIPTION
**Pull request by fhenrich33**

### CODE REVIEW

**Comment by fhenrich33** [comment]
[2.0 BUG: Incorrect "no access to camera" errors are spamming the user repeatedly.](https://trello.com/c/EkeQn7EJ/1471-20-bug-incorrect-no-access-to-camera-errors-are-spamming-the-user-repeatedly)

**Comment by fhenrich33** [comment]
[Bug: If the QR Scanner doesn't have access to the camera, there should be an error message and an ability for users to re-request.](https://trello.com/c/CHbAL8Ha/1420-bug-if-the-qr-scanner-doesnt-have-access-to-the-camera-there-should-be-an-error-message-and-an-ability-for-users-to-re-request)

File: front/src/components/QrReader/QrReaderContainer.tsx [hunk 1]
```diff
@@ -46,21 +50,23 @@ function QrReaderContainer({ onSuccess }: IQrReaderContainerProps) {
...
-  }
+      .getUserMedia({
+        audio: false,
+        video: true,
+      })
+      .catch((error) => {
+        if (error.name === "NotAllowedError") {
+          setIsCameraNotPermited(true);
+        } else {
+          createToast({
```
Comment (HaGuesto) [inline front/src/components/QrReader/QrReaderContainer.tsx:61]: pls change this back

File: front/src/components/QrReader/QrReaderContainer.tsx [hunk 2]
```diff
@@ -153,24 +159,29 @@ function QrReaderContainer({ onSuccess }: IQrReaderContainerProps) {
 
   useEffect(() => {
     checkCameraPermission();
-  })
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (HaGuesto) [inline front/src/components/QrReader/QrReaderContainer.tsx:162]: change rule maybe in DX PR

File: front/src/components/Alerts.tsx [hunk 3]
```diff
@@ -3,16 +3,18 @@ import { Alert, AlertIcon, chakra, Stack } from "@chakra-ui/react";
 
 export interface IAlertWithoutActionProps {
   alertText: React.ReactNode;
+  // eslint-disable-next-line react/require-default-props
+  type?: "error" | "warning";
```
Comment (HaGuesto) [inline front/src/components/Alerts.tsx:7]: change rule maybe in DX PR

**Review by HaGuesto — COMMENTED** [review]
## code review done

**Review by HaGuesto — APPROVED** [review]

**Pull request merged**